ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature

```diff
@@ -43,4 +43,16 @@ Scenario: Representative completed checklist and self test redirected to Video W
 Scenario: Judge redirected to Video Web
 	Given a new browser is open for a Judge
 	When the user logs in with valid credentials
+	Then the user is redirected to Video Web
+
+@VIH-4577 @PanelMember
+Scenario: Panel member redirected to Video Web
+	Given a new browser is open for a Panel Member
+	When the user logs in with valid credentials
+	Then the user is redirected to Video Web
+
+@VIH-4577 @Winger
+Scenario:  Winger redirected to Video Web
```

Comment:
@valentinabennett  could you add a test here to make sure JOH user role gets re-directed to v.web as well. That way it covers any other hearings roles in the future that have the JOH user role

Reply:
Thats going to be difficult from here though @RuwanikaWeerakkody, and the UserRole of JOH is on the conference, and the test setup will only know their user type (winger, panel member, individual etc...)

Reply:
we have user-type, I don't think I could do it with user role ( winger required special case type)

Reply:
user-role we get by user-type, currently we could not do by user-role